platforms/documentation/docs/src/snippets/bestPractices/useConventionPlugins/kotlin/avoid-this/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+// tag::avoid-this[]
+plugins {
+    java
+}
+
+// Duplicated configuration across multiple build files
```

Comment:
I think it might be better to demo separate Good and Bad builds, so each could be a multi-project build and better distinguish the 2 approaches.  Understanding what parts of this build are good and bad isn't completely clear now.

Reply:
Ok, I'll separate them.